### DISCUSSION TITLE
Cambio Modem

### DISCUSSION THREAD

**Discussion by TheSkull00** [indomus/forum#215] [HUB: Home Assistant]
Salve a tutti. Ho hassio installato su raspberry pi 4 collegato via cavo LAN. Ho cambiato operatore di linea fissa e di conseguenza anche il modem. Ho configurato ip statico e regole port forwarding sul nuovo modem nello stesso modo in cui avevo configurato anche il vecchio modem. Il problema è che adesso non riesco più ad accedere ad hassio in nessun modo ne in locale e tantomeno in remoto. Qualche consiglio?

**Comment by 50494554524F** [comment]
sarai passato a fastweb che ha rete nattata... devi chiamarli e farti dare ip pubblico

**Reply by TheSkull00** [reply]
sono passato a eolo e ho ip pubblico attivato. Mi sono dimenticato un particolare importante. L'impossibilità di accedere ad hassio riguarda solo lovelace sia remoto che locale. Mentre per quanto riguarda accesso ftp o ssh in locale riesco tranquillamente ad accedere ad entrambi.

**Reply by snakuzzo** [reply]
Sei autonomo su apertura porte?

**Reply by TheSkull00** [reply]
Dal modem posso aprire le porte tranquillamente, se è questo che intendi.
